helpers/password_helper.go

```diff
@@ -4,9 +4,15 @@ import "golang.org/x/crypto/bcrypt"
 
 // HashPassword hashes plain password with bcrypt adaptive hashing algorithm.
 func HashPassword(plainPassword string) (hashedPassword string, err error) {
-	bytePassword := []byte(plainPassword)
-	hashPassword, err := bcrypt.GenerateFromPassword(bytePassword, bcrypt.DefaultCost)
+	hashPassword, err := bcrypt.GenerateFromPassword([]byte(plainPassword), bcrypt.DefaultCost)
 	hashedPassword = string(hashPassword)
 
 	return hashedPassword, err
 }
+
+// CheckMatchPassword checks match password between plain password and hashed password
+func CheckMatchPassword(hashedPassword string, plainPassword string) (err error) {
```

Comment:
Could we do it like? 
```suggestion
func CheckMatchPassword(hashedPassword, plainPassword string) (err error) {
```

Reply:
@junan Could you give me more detail for this changing?
I think it looks fine for me as we specific to the input data type 🤔

Reply:
@junan @Lahphim Regarding [the convention](https://nimblehq.co/compass/development/code-conventions/golang/#functions) we should declare the type of each parameter separately 💭